python/tests/test_tables.py

```diff
@@ -2272,32 +2272,30 @@ class TestSubsetTables(unittest.TestCase):
 
     seed = 12345
 
-    def get_msprime_mig_example(self, N=100, T=100, n=10, mutations_per_branch=0):
-        M = [[0.0, 0.0], [0.0, 0.0]]
-        population_configurations = [
-            msprime.PopulationConfiguration(sample_size=n),
-            msprime.PopulationConfiguration(sample_size=n),
-        ]
-        demographic_events = [
-            msprime.MassMigration(T, source=1, dest=0, proportion=1),
-            msprime.CensusEvent(time=T),
-        ]
-        ts = msprime.simulate(
-            Ne=N,
-            population_configurations=population_configurations,
-            demographic_events=demographic_events,
-            migration_matrix=M,
-            length=2e4,
-            recombination_rate=1e-8,
-            mutation_rate=1e-8,
-            record_migrations=True,
-            random_seed=self.seed,
-        )
-        if mutations_per_branch > 0:
-            ts = tsutil.insert_branch_mutations(ts, mutations_per_branch)
-        return ts
-
-    def verify_subset_equality(self, tables, subset, nodes):
+    def verify_subset_equality(self, tables, nodes):
```

Comment:
this is verifying both implementations agree; probably we will stick with the C version, and will put the numpy version in here, to compare against.